novu/api/topic.py

```diff
@@ -0,0 +1,109 @@
+"""
+This module is used to define the ``TopicApi``, a python wrapper to interact with ``Topics`` in Novu.
+"""
+from typing import Dict, List, Optional, Tuple, Union
+
+from novu.api.base import Api
+from novu.constants import TOPICS_ENDPOINT
+from novu.dto.topic import PaginatedTopicDto, TopicDto
+
+
+class TopicApi(Api):
+    """This class aims to handle all API methods around topics in API"""
+
+    def __init__(self, url: Optional[str] = None, api_key: Optional[str] = None) -> None:
+        super().__init__(url, api_key)
+
+        self._topic_url = f"{self._url}{TOPICS_ENDPOINT}"
+
+    def list(
+        self, page: Optional[int] = None, limit: Optional[int] = None, key: Optional[str] = None
+    ) -> PaginatedTopicDto:
+        """List existing topics
+
+        Args:
+            page: Page to retrieve. Defaults to None.
+            limit: Size of the page to retrieve. Defaults to None.
+            key: Filter list by a topic key. Defaults to None.
+
+        Returns:
+            Paginated list of topic
+        """
+        payload: Dict[str, Union[int, str]] = {}
+        if page:
+            payload["page"] = page
+        if limit:
+            payload["limit"] = limit
+        if key:
+            payload["key"] = key
+
+        return PaginatedTopicDto.from_camel_case(self.handle_request("GET", f"{self._topic_url}", payload=payload))
```

Comment:
```suggestion
        return PaginatedTopicDto.from_camel_case(self.handle_request("GET", self._topic_url, payload=payload))
```
C'est déjà un `str`, non?

Reply:
Présent à d'autres endroits aussi